scene/resources/particle_process_material.cpp

```diff
@@ -774,7 +777,7 @@ void ParticleProcessMaterial::_update_shader() {
 	code += "	}else{radial_displacement = get_random_direction_from_spread(alt_seed, 360.0)* param.radial_velocity;} \n";
 	code += "	if (radial_displacement_multiplier * param.radial_velocity < 0.0){\n // Prevent inwards velocity to flicker once the point is reached.";
 	code += "		if (length(radial_displacement) > 0.01){\n";
-	code += "		radial_displacement = normalize(radial_displacement) * min(abs((radial_displacement_multiplier * param.radial_velocity)), length(transform[3].xyz - global_pivot));\n";
+	code += "		radial_displacement = normalize(radial_displacement) * min(abs((radial_displacement_multiplier * param.radial_velocity)), length(transform[3].xyz - global_pivot) / delta);\n";
```

Comment:
I'm a bit skeptical of this change. Prior to https://github.com/godotengine/godot/pull/83488, the `delta` was multiplied by `radial_displacement`, then it was removed, now `radial_displacement` is divided by `delta`. Which is reversing the behaviour from what it was previously. 

Can you provide a little bit more context why dividing is now needed instead of multiplying?

Reply:
I tried multiplying first and it didn't work so I assumed my math assumptions were wrong

Reply:
This calls for dimensional analysis :)
But that won't be easy with all these variable names which are unclear about whether they're distances, velocities, speeds, distance deltas, or accelerations :/